### PR TITLE
Color doesn't have a fourth argument

### PR DESCRIPTION
**Pull request by gero3**
This fixes [lgtm error](https://lgtm.com/projects/g/mrdoob/three.js/snapshot/b9df354c311a1aaab5e3aa6f5c91d625aa6ced49/files/examples/jsm/loaders/AssimpLoader.js?sort=name&dir=ASC&mode=heatmap#xcf576678bc239e7c:1)

### CODE REVIEW

**Review by Mugen87 — DISMISSED** [review]

File: examples/jsm/loaders/AssimpLoader.js [hunk 1]
```diff
@@ -878,7 +878,7 @@ AssimpLoader.prototype = {
 			this.a = 0;
 			this.toTHREE = function () {
 
-				return new Color( this.r, this.g, this.b, 1 );
+				return new Color( this.r, this.g, this.b );
```
Comment (Mugen87) [inline examples/jsm/loaders/AssimpLoader.js]: see https://github.com/mrdoob/three.js/pull/16572#discussion_r287752468

Comment (Mugen87) [reply examples/jsm/loaders/AssimpLoader.js]: BTW: This line has the same error: https://github.com/mrdoob/three.js/pull/16574/files#diff-0f9644de66075bc5c997fbc470225584R867

**Comment by jxv106** [comment]
你好




------------------ Original ------------------

File: examples/jsm/loaders/AssimpLoader.js (hunk 1 above)
Comment (gero3) [reply examples/jsm/loaders/AssimpLoader.js]: yeah but there alpha isn't ensured to be equal to 1. So i didn't know how to solve that one yet

Comment (gero3) [reply examples/jsm/loaders/AssimpLoader.js]: I've edited the PR to remove both errors.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**